mobile-widgets/qml/Export.qml

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: GPL-2.0
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.12
+import org.subsurfacedivelog.mobile 1.0
+import org.kde.kirigami 2.4 as Kirigami
+
+Kirigami.ScrollablePage {
+	title: qsTr("Export Divelog information")
```

Comment:
while adding a dummy page might make sense in your logical step by step process to develop this code, when I read through the PR to review things this feels odd. Why don't you just have the final Export.qml page right here - or at least the skeleton with all the UI elements (I haven't read all the way to the end, yet, just in case there are interfaces that the final page will use that aren't implemented yet - you could comment them out at this stage).
But adding a dummy page that is filled out a few commits later seems... odd - at least when adding a new feature like this.